make test-links fail when hugo's docsy theme is not installed

## Feature Request

When a user tries to execute `make test-links` and has not installed hugo's docsy theme the task fails with:

```
msoriano@localhost:~/go/src/github.com/miguelsorianod/operator-sdk (add-missing-makefile-bundle-prerequisite)$ make test-links
./hack/check-links.sh
Building the site and checking links
sdk-html
Building sites … Error: Error building site: "/src/website/content/en/_index.html:5:1": failed to extract shortcode: template for shortcode "blocks/cover" not found
Total in 125 ms
make: *** [Makefile:103: test-links] Error 255
```

Specifically it seems to fail in the following line of the `hack/check-links.sh` script:
```
docker run --rm -v "$(pwd):/src" -v sdk-html:/src/website/public klakegg/hugo:0.73.0-ext-ubuntu -s website
```

It is true that in operator-sdk documentation https://sdk.operatorframework.io/docs/contribution-guidelines/documentation/#prerequisites a prerequisite is to install it by updating/installing the submodules to install the theme with:
```
git submodule update --init --recursive
```

However, I wonder if if would be useful/possible to automate this somehow? As a prerequisite executing the line above in the corresponding Makefile tasks? However, I'm not entirely sure if that approach would be the appropriate one if executed unconditionally (as it would always update the contents each time that make target would be executed?)

What do you think?

Thank you.

--- a/Makefile
+++ b/Makefile
@@ -110,6 +110,7 @@ test-sanity: generate fix ## Test repo formatting, linting, etc.
 
 .PHONY: test-links
 test-links: ## Test doc links
+	git submodule update --init --recursive website/
 	./hack/check-links.sh
 
 .PHONY: test-unit

